spec/lucky/input_helpers_spec.cr

```diff
@@ -215,6 +219,10 @@ describe Lucky::InputHelpers do
     view.textarea(form.first_name, class: "cool").to_s.should contain <<-HTML
     <textarea id="user_first_name" name="user:first_name" class="cool">My name</textarea>
     HTML
+
+    view.textarea(form.first_name, rows: 5, cols: 15).to_s.should contain <<-HTML
+    <textarea id="user_first_name" name="user:first_name" rows="5" cols="15">My name</textarea>
+    HTML
   end
 end
 
```

Comment:
Could you add some specs for links to just to make sure they work for those? They work a bit differently so I’m not sure if they’ll work out of the box.

Reply:
When you say "links", do you mean the `link(to: LinkHelpers::Index, class: "link") { }` method? Like maybe `link(to: Whatever, "data-something": true)`?